### ISSUE TITLE
[Bug] Track design placement completely bugged

### ISSUE THREAD

**Issue by frutiemax** [OpenRCT2/OpenRCT2#12261]
Open the Create your own Magic Mountain park, then try to place a corkscrew roller coaster. The roller coaster preview is not drawn correctly. Also, the track design when placed is completely bugged...

I am on commit c8b66dd2894b4a7fc3f6403139553bedb86d255f.

**Comment by Gymnasiast** [comment]
I'm on the same commit, but I cannot reproduce this bug.

**Comment by frutiemax** [comment]
Yeah I just tried to bisect and I couldn't reproduce the bug either...

**Comment by Gymnasiast** [comment]
Are you sure you were on c8b66dd and not on an earlier commit?
